src/neuroglancer/sliceview/volume/renderlayer.ts

```diff
@@ -433,6 +433,14 @@ export class RenderLayer extends GenericSliceViewRenderLayer {
     return this.gl.memoize.get(key, () => this.buildShader());
   }
 
+  setGLBlendMode(gl: WebGLRenderingContext, renderLayerNum: number) {
```

Comment:
If you define this in sliceview/renderlayer.ts, you don't have to duplicate it for both vector_graphics and volume.